plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -516,7 +516,7 @@ public static function register_post_types() {
 											'woocommerce/product-schedule-sale-fields',
 										),
 										array(
-											'woocommerce/product-radio',
+											'woocommerce/product-radio-field',
```

Comment:
Just thinking out loud here, some of these more generic blocks don't *need* to be coupled to a product, as they would work elsewhere as well. Since we're operating in the `product-editor` package this may be a moot point for now, and it will help prevent conflicts to leave the `product-` portion, but worth keeping in mind.

Reply:
Yep, I was thinking on that too. `collapsible` `tab` `radio` `checkbox` are some examples. We should talk about that to see what should be the right name for these generic ones. Then only concern I have is that whatever the name convention we pick should not be having a short name if we want to prevent collisions in the future, so it should be something longer or difficult to collide.